app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="text_main_activity_margin_start">15dp</dimen>
+    <dimen name="text_main_activity_margin_end">15dp</dimen>
```

Comment:
if the margin is the same, also you can use a horizontal margin

Reply:
done